src/docfx/config/ops/DocsEnvironment.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#nullable enable
+
+namespace Microsoft.Docs.Build
+{
+    public enum DocsEnvironment
+    {
+        Production,
```

Comment:
should it be `Prod`, `PPE` then?

Reply:
This is only internal name and I think it's OK to have Production and Sandbox which is same with before.

Reply:
I mean will `Enum.TryParse` fail?